apps/profile/apps.py

```diff
@@ -0,0 +1,5 @@
+from django.apps import AppConfig
+
+
+class ServicesConfig(AppConfig):
```

Comment:
What's the use of this?

Reply:
It helps us add the profiles applications for introspection